components/fullscreen-image-quote/styles.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable security/detect-object-injection */
+import { cnb } from "cnbuilder";
+
+export const quoteVAligns = {
+  top: "lg:su-items-start",
+  center: "lg:su-items-center",
+  bottom: "",
+};
+
+export const quoteHAligns = {
+  left: "su-ml-0 su-mr-auto lg:su-pr-0",
+  right: "su-ml-0 su-mr-auto lg:su-mr-0 lg:su-ml-auto lg:su-pl-0",
+};
+
+export const imageVPositions = {
+  top: "su-object-top",
+  center: "su-object-center",
+  bottom: "su-object-bottom",
+};
+
+export const root = (removeTopSpacing) =>
+  cnb(
+    "su-aspect-w-1 su-aspect-h-2 sm:su-aspect-w-3 sm:su-aspect-h-4 lg:su-aspect-h-2 2xl:su-aspect-w-2 2xl:su-aspect-h-1 su-bg-black-true su-text-white",
```

Comment:
@broleomo @iamrentman I made some dev choices about aspect ratio of the component here. These seem to be working out best so far. I try to match the Figma aspect ratios as much as possible, while avoid having the component be so large that one would have to scroll up and down to see the image and the content. Also, I'm picking reasonable integers for the aspect ratios, e.g., 2x1 instead of something like 49x105 😄  The choices currently are:

Phone (XS): 1x2
Small and regular tablets (SM-MD): 3x4
Large tablets and small desktop (LG-XL): 3x2
Most common desktop (2XL and up): 2x1

Reply:
Thanks for considering this. Your choices are 💯! 

Also, love this way of adding the styles as an import to the component...

Reply:
Thanks, Justin! Yeah there are quite a number of conditionals for styles in this component. I put the classes in a separate file so it's more organized and a bit easier to read.